lib/model/ride_attendee_scanning/ride_attendee_scanning_delegate.dart

```diff
@@ -297,6 +276,41 @@ class RideAttendeeScanningDelegate {
     _rideAttendeeController.add(currentSelection);
   }
 
+  /// Resolve the list of possible owners for the given [device] and add the device to the list of found devices.
+  ///
+  /// If the device has one owner, its owner is added to the list of scanned ride attendees.
+  /// If the device has multiple possible owners,
+  /// the list of possible owners for [device] is added to the list of unresolved device owners.
+  void _resolvePossibleOwnersForDeviceAndAddDeviceToFoundDevicesList(BluetoothPeripheral device) {
```

Comment:
The old name of this method, `_addScannedDevice` is misleading. Since we now have a formatting line length of 120 characters, I rectified it.